assets/shared/components/stepper/check-icon.js

```diff
@@ -0,0 +1,31 @@
+export default () => (
+	<svg
+		role="img"
+		aria-hidden="true"
+		focusable="false"
+		width="18"
+		height="18"
+		viewBox="0 0 18 18"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+	>
+		<mask
+			id="mask0"
+			mask-type="alpha"
```

Comment:
I think that this doesn't do anything? Should we replaced with a style?

Reply:
This is directly from WC's `check-icon.js` so I'm not sure about the original intent. If you understand it I could remove it, but is it hurting anything?

Reply:
No that's fine. I was just asking in case this this was meant to achieve some effect.